internal/zinc-persist/src/main/scala/sbt/internal/inc/text/TextAnalysisFormat.scala

```diff
@@ -36,6 +36,63 @@ class TextAnalysisFormat(val mappers: ReadWriteMappers)
   import sbinary.Format
   import xsbti.{ Position, Problem, Severity }
 
+  // TODO: Remove this method once a version of sbinary containing
```

Comment:
You don't need to update the text analysis format since it's not supported anymore and it's not used by any build tool that I know of (and if somebody is using it, it's fine that they don't see the new additions to positions). We should probably remove this from this PR

Reply:
> You don't need to update the text analysis format since it's not supported anymore 

Then why isn't it deprecated? If it was, I wouldn't have wasted a day implementing this.

 > We should probably remove this from this PR

At this point I would just keep it since it avoids some deprecation warnings in the build of Zinc.